src/controllers/incorrect-information/wrong.officer.details.controller.ts

```diff
@@ -1,15 +1,18 @@
 import { Request, Response } from "express";
-import { urlUtils } from "../../utils/url";
 import { ACTIVE_OFFICERS_DETAILS_PATH, TASK_LIST_PATH } from "../../types/page.urls";
 import { Templates } from "../../types/template.paths";
-import { WRONG_DETAILS_UPDATE_OFFICER, WRONG_DETAILS_UPDATE_OFFICERS } from "../../utils/constants";
+import {
+  WRONG_DETAILS_UPDATE_OFFICER,
+  WRONG_DETAILS_UPDATE_OFFICERS
+} from "../../utils/constants";
+import { buildWrongDetails } from "../../services/wrong.details.service";
 
 export const get = (req: Request, res: Response) => {
-  return res.render(Templates.WRONG_DETAILS, {
-    templateName: Templates.WRONG_DETAILS,
-    backLinkUrl: urlUtils.getUrlToPath(ACTIVE_OFFICERS_DETAILS_PATH, req),
-    returnToTaskListUrl: urlUtils.getUrlToPath(TASK_LIST_PATH, req),
-    stepOneHeading: WRONG_DETAILS_UPDATE_OFFICER,
-    pageHeading: WRONG_DETAILS_UPDATE_OFFICERS,
-  });
+  return res.render(Templates.WRONG_DETAILS, buildWrongDetails(
```

Comment:
i don't think the buildWrongDetails makes much of an improvement. I prefer it as it was as we can see what fields are being sent to the template without having to click out of the controller into the buildWrongDetails. See what @mwestacott thinks.

Reply:
Sonar was complaining about duplication.

Reply:
Removed new service and put the controllers back to how they were.